src/main/java/com/exadel/discount/service/impl/JwtServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.exadel.discount.service.impl;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.exadel.discount.service.JwtService;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+@Setter
+@Getter
+@Slf4j
+public class JwtServiceImpl implements JwtService {
```

Comment:
add all methods from the JwtGenerationServiceImpl here and add additional implements for second interface - JwtGenerationService

Reply:
Already mentioned in previous PR: https://github.com/SergeyKoval/2021_june_team4_be/pull/28#discussion_r661245118

Reply:
This was resolved in the commit "87b09a6".